lib/Syntax/CMakeLists.txt

```diff
@@ -4,15 +4,17 @@ else()
   set(SWIFT_GYB_FLAGS --line-directive "\'#line" "%(line)d" "\"%(file)s\"\'")
 endif()
 
-add_swift_library(swiftSyntax STATIC
-  SyntaxNodes.cpp.gyb
-  SyntaxBuilders.cpp.gyb
-  SyntaxKind.cpp.gyb
-  SyntaxFactory.cpp.gyb
-  SyntaxVisitor.cpp.gyb
-  Trivia.cpp.gyb
-  RawSyntax.cpp
-  Syntax.cpp
-  SyntaxData.cpp
-  SyntaxSerialization.cpp.gyb
-  UnknownSyntax.cpp)
+add_swift_library(swiftSyntax
+                  STATIC
```

Comment:
Nitpick: `STATIC` is the type of library, not a delimiter marking normal sources.

Reply:
True.  I can put that on the previous line, or unindent the source list.  Don't really care too much between the two.  I think that having the options on their own line is easier to find though.